### ISSUE TITLE
wpml_sync_all_custom_fields action

### ISSUE THREAD

**Issue by dylantuohy** [joppuyo/acf-image-aspect-ratio-crop#58]
Hello, thanks for the nice plugin.

My goal is to copy the alt text of the original image to the new cropped image. I found the `wpml_sync_all_custom_fields` action called at the end of the `create_crop` function, and I'm using it to do this:

```php

add_action( 'wpml_sync_all_custom_fields', 'dt_aiarc_copy_alt');
function dt_aiarc_copy_alt( $attachment_id ) { 
 
  // only proceed if this is an image
  if (!wp_attachment_is_image($attachment_id)){
    return;
  }

  // get image meta
  $img_meta = get_post_meta($attachment_id);
  
  // if this image has an "aiarc" original, but does not have alt text
  if ( isset($img_meta['acf_image_aspect_ratio_crop_original_image_id']) && !isset($img_meta['_wp_attachment_image_alt'])){

    // get original image
    $original_image = get_post_meta($img_meta['acf_image_aspect_ratio_crop_original_image_id'][0]);

    // if original image has alt text...
    if ( isset($original_image['_wp_attachment_image_alt']) ){
      
      // copy the original's alt text to the cropped image
      add_post_meta( $attachment_id, '_wp_attachment_image_alt', $original_image['_wp_attachment_image_alt'][0] );
    
    }

  }
 
}
```

This is working OK but I have some questions.

- I realize `wpml_sync_all_custom_fields` is related to the WPML plugin but I don't fully understand what it does. Could this cause problems on a site with WPML installed?
- Is there a better way to do this (copy alt text from original image)?
- If not, any chance this feature will be added in the future as a field setting?
